assets/docs/rishit-dagli/plant-disease/default/tfjs/1.md

```diff
@@ -0,0 +1,30 @@
+# Tfjs rishit-dagli/plant-disease/default/tfjs/1
+TF.js deployment of rishit-dagli/plant-disease/1 .
+
+<!-- parent-model: rishit-dagli/plant-disease/1 -->
+<!-- asset-path: https://storage.googleapis.com/rishit-dagli/plant-disease-model/tfjs_model.tar.gz -->
+
+[![Open Colab notebook]](https://colab.research.google.com/github/Rishit-dagli/Greenathon-Plant-AI/blob/main/notebooks/plant_diseases_TFJS.ipynb)
+
+### Origin
+This model is based on [rishit-dagli/plant-disease/1](https://tfhub.dev/rishit-dagli/plant-disease/1/).
+
+This model has been automatically converted using the [TF.js converter](https://github.com/tensorflow/tfjs/tree/master/tfjs-converter).
+
+### Overview
+This model classifies plant leaf images into 38 categories by species and disease. This model was trained on the [plant_village](https://www.tensorflow.org/datasets/catalog/plant_village) (Hughes et al.) dataset [1] on about 87,000 (+ augmented images) healthy and unhealthy leaf images. This model was trained on top of the MobileNet model using weights from MobileNet trained on ImageNet. More info about using this model could be found in the below section.
+
+Here are a couple of images from the dataset on which this model was trained on-
+
+![](https://github.com/Rishit-dagli/Greenathon-Plant-AI/blob/main/media/dataset-examples.png)
+
+### Example use
+This model can be loaded using TF.js as:
+
+```js
+tf.loadLayersModel("https://tfhub.dev/rishit-dagli/plant-disease/default/tfjs/1", { fromTFHub: true })
```

Comment:
This model will actually be available at https://tfhub.dev/rishit-dagli/tfjs-model/plant-disease/default/tfjs/1.

Reply:
Got it 👍 Thanks for pointing this out!